PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -197,11 +197,31 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	//void NPPM_TRIGGERTABBARCONTEXTMENU(int view, int index2Activate)
 
 	#define NPPM_GETNPPVERSION (NPPMSG + 50)
-	// int NPPM_GETNPPVERSION(0, 0)
-	// return version
-	// ex : v4.6
-	// HIWORD(version) == 4
-	// LOWORD(version) == 6
+	// int NPPM_GETNPPVERSION(BOOL ADD_ZERO_PADDING, 0)

```

Comment:
OK, but how does a plugin use this feature?
You have to first discover the current version &#x2014; by calling `NPPM_GETNPPVERSION`!
If the running N++ has the new feature, you call `NPPM_GETNPPVERSION` a second time, with the extra parameter . . . ?

Reply:
The best way IMO is to check number of digit in LOWORD - if it's not 3 digits, then it's for plugin to do the zero padding.
I have no better solution so far.